CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Remove service name from `otelmongodb` configuration and span attributes. (#763)
 
+### Fixed
+
+- Fix deadlocks and race conditions in Sarama instrumentation. (#754)
```

Comment:
Should we mention that MessageID and KafkaPartitionID attributes are not set if a message was not processed?

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-go-contrib/pull/754/commits/341acf7b775884862866a492c182004232cf2d8d